src/components/ProgressForm/ProgressLayout.tsx

```diff
@@ -20,12 +20,10 @@ const ProgressLayout: FC = () => {
     return (
         <>
             <NavBar />
-            <div className={styles.banner}>
-                <ProgressHeader
-                    totalMissedUpdates={totalMissedUpdates}
-                    updateType="Progress"
-                />
-            </div>
+            <ProgressHeader
+                totalMissedUpdates={totalMissedUpdates}
```

Comment:
Are we handling isNaN scenarios here?

Reply:
yes, we are handling this as if the date array length is 0 so it should so 0, and inside the array we have restricted array element type to be numbers. so I guess NaN situation won't occur